src/js/_enqueues/wp/embed.js

```diff
@@ -11,18 +11,15 @@
 (function ( window, document ) {
 	'use strict';
 
-	var supportedBrowser = false,
-		loaded = false;
-
-		if ( document.querySelector ) {
-			if ( window.addEventListener ) {
-				supportedBrowser = true;
-			}
-		}
+	// Abort for ancient browsers.
+	if ( ! document.querySelector || ! window.addEventListener || typeof URL === 'undefined' ) {
+		return;
+	}
```

Comment:
Ugh, the minifier is injecting `&&` here:

```
!function(d,l){"use strict";l.querySelector&&d.addEventListener&&"undefined"!=typeof URL&&...
```

This is breaking the `verify:wp-embed` check.

Reply:
IIRC that's why all those checks were separate :)

Reply:
@swissspidy Given that [r35708](https://core.trac.wordpress.org/changeset/35708) was 8 years ago, would it still be a concern that old versions of WP convert `&` to `&#038;`?

Reply:
That commit was for WordPress 4.4, so older versions would be 4.3 and below. 

![image](https://github.com/WordPress/wordpress-develop/assets/134745/5aa1a3eb-49ef-4ffe-af1a-70cfd002b70a)

Versions of WordPress under 4.4 account for 1.43% of all installs, so I think we've safely moved beyond the need for this:

![image](https://github.com/WordPress/wordpress-develop/assets/134745/26faf2ba-8e39-4d6a-823a-a999d50261a0)

Reply:
I've gone ahead and removed the anti-ampersand stipulation.

Reply:
Good question! So that was for sites running WP 4.3 or older (without https://core.trac.wordpress.org/changeset/35709), if I am not mistaken. It should be fine to drop that by now. But maybe in a separate ticket?

Reply:
Opened another ticket: https://core.trac.wordpress.org/ticket/58974